packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -505,6 +520,35 @@ export default {
             }
         },
 
+        /**
+         * Look up the geo details for the customers address, skip on failure.
+         *
+         */
+        async lookupGeoLocation () {
+            const postData = { addressLines: [] };
+            Object.keys(this.address).forEach(propName => {
+                if (this.address[propName]) {
+                    postData.addressLines.push(this.address[propName]);
+                }
+            });
+
+            try {
+                if (postData.addressLines.length) {
```

Comment:
You could move this check to the vuex action instead, where the actual axios call is happening. Just means the async call (Vuex wise) will still happen but the actual axios won't. Also it would be easier, if it's required later, to commit a different type of state update if we wanted to display a message or $emit a value up to coreWeb if that state was anything but lat/longs populated when address lines aren't populated etc.

Reply:
Discussed and leaving as is 👍